src/components/metadata/metadata.test.js

```diff
@@ -20,6 +22,8 @@ describe('MetaData', () => {
       afterLayoutActions: [
         // Click the expected node
         () => toggleNodeClicked(props.nodeId),
+        //simulating loadNodeData in node.js
```

Comment:
rewording this to `simulating loadNodeData action in 'action/node.js'` would be more explicit